pkg/kubectl/pdb.go

```diff
@@ -51,15 +50,15 @@ func (s PodDisruptionBudgetV1Generator) Generate(params map[string]interface{})
 	}
 	name, isString := params["name"].(string)
 	if !isString {
-		return nil, fmt.Errorf("expected string, saw %v for 'name'", name)
+		return nil, fmt.Errorf("expected string, found %v for 'name'", params["name"])
```

Comment:
I'm wondering if we should use `%#v`, since this error happens because the type is wrong. But yeah good catch.

Reply:
I agree with what you're saying, but instead of `%#v`, I believe it will be better to use `%T` instead. The error message will contain the type representation: `expected string, found int for ...`.